clang/test/CIR/CodeGen/three-way-comparison.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c++20 -fclangir-enable -clangir-disable-passes -emit-cir %s -o - | FileCheck %s -check-prefix=CIRGEN
```

Comment:
Please use the before / after printers as done in `static.cpp`

Reply:
Updated. Cool.